lib/pterodactyl_package.php

```diff
@@ -346,14 +346,13 @@ public function getRules(array $packageLists, array $vars)
             'meta[port_range]' => [
                 'format' => [
                     'rule' => function ($portRanges) {
-                        if (!empty($portRanges)) {
-                            $ranges = explode(',', $portRanges);
-                            foreach ($ranges as $range) {
-                                if (!preg_match('/^[0-9]+\-[0-9]+$/', $range)) {
-                                    return false;
-                                }
+                        $ranges = explode(',', $portRanges);
+                        foreach ($ranges as $range) {
+                            if (!preg_match('/^[0-9]+\-[0-9]+$/', $range)) {
+                                return false;
                             }
                         }
+
                         return true;
```

Comment:
It looks like this returns true when no port range is provided at all.
I would expect this line to `return !empty($ranges);`

Reply:
$ranges is never empty.  Explode will return an array with 1 item that is an empty string if the delimiter does not exist in the port range.

Reply:
Well, there are certain circumstances that `explode` will return `false`. But as long as providing no value will work correctly with this functionality, that is fine.